tests/expense.test.js

```diff
@@ -10,6 +11,7 @@ describe('Expense', () => {
   beforeEach(() => {
     server = createServer();
     api = supertest(server);
+    expenses.length = 0;
```

Comment:
Directly modifying the length of expenses array to 0 can lead to unintended behavior.